lib.php

```diff
@@ -33,3 +33,63 @@
 // Media type constants.
 define('LOCAL_SMARTMEDIA_TYPE_AUDIO', 'Audio');
 define('LOCAL_SMARTMEDIA_TYPE_VIDEO', 'Video');
+
+/**
+ * Serve the files from the local smartmedia file areas.
+ *
+ * @param stdClass $course The course object.
+ * @param stdClass $cm The course module object.
+ * @param stdClass $context The context.
+ * @param string $filearea The name of the file area.
+ * @param array $args Extra arguments (itemid, path).
+ * @param bool $forcedownload Whether or not force download.
+ * @param array $options Additional options affecting the file serving.
+ * @return bool False if the file not found, just send the file otherwise and do not return anything.
```

Comment:
Would it not be better to return `true` even if we aren't handling it at the moment, just for consistency?

Reply:
Nope, this is old school Moodle code. The sendfile method that is called eventually calls die; when the file is sent. So the final return true will never actually be reached.